roles/test_sensubility/tasks/test_health_status.yml

```diff
@@ -1,8 +1,8 @@
   #In this test we choose one container (logrotate_crond),checking that it is running 
   #and then stopping the container and starting it again and checking that its health status changes accordingly.
 
-- name: RHELOSP-176011 Check that logrotate_crond container is running on controller-0
-  delegate_to: "{{ groups['overcloud_nodes'][0] }}"
+- name: RHELOSP-176011 Check that logrotate_crond container is running on ceph-0
+  delegate_to: ceph-0
```

Comment:
All of these are delegated to ceph-0. Does it matter what host it runs on, or can it run on any host?

It might make more sense to remove the delegate_to and update the stf playbook to add a play that runs on the ceph node for this. Could this also run on the controller?

Reply:
yes,it can run on all overcloud nodes. They all have that container